exonum/src/blockchain/tests.rs

```diff
@@ -93,9 +93,9 @@ fn test_correct_encoding_struct() {
     test.write(&mut buffer, 0, 8);
     assert_eq!(buffer, dat);
     <StructWithTwoSegments as Field>::check(&dat, 0.into(), 8.into(), 8.into()).unwrap();
-    let strukt = unsafe { <StructWithTwoSegments as Field>::read(&dat, 0, 8) };
-    assert_eq!(strukt.first(), &[1u8]);
-    assert_eq!(strukt.second(), &[2u8]);
+    let _struct = unsafe { <StructWithTwoSegments as Field>::read(&dat, 0, 8) };
```

Comment:
I don't like underscore prefixes because they suppress "unused variable" warnings. I propose to use `struct_` if some context-meaningful name cannot be used.

Reply:
Fair enough